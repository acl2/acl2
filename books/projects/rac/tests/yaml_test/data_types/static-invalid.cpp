// RAC begin

int foo() {
  // This must not be accepted by the parser since it add state to the
  // function.
  static int foo = 3;
  return 0;
}
