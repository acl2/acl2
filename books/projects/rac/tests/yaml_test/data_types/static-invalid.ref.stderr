static-invalid.cpp:6 (9-12): syntax error, unexpected INT, expecting CONST
 6 |   static int foo = 3;
   |          ^^^
